test/extended/prometheus/statefulsets_stability.go

```diff
@@ -0,0 +1,60 @@
+package prometheus
```

Comment:
I assume this will also apply to old upgrades that we still test, don't know if we had statefulsets recreation back then.

Reply:
The latter I believe though it might not be true for jobs that test multiple upgrades (like https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-upgrade-4.11-to-4.12-to-4.13-to-4.14-ci)? :thinking:

Reply:
Maybe in `Skip()` below you can inspect the UpgradeContext and skip accordingly?

Reply:
Good idea Jan, we can get that from `upgCtx.Versions[*].Version`, but I will ask around to make sure we really need this.

Reply:
I did some digging, IIUC the whole upgrade chain is considered as a single disruption, so we only run upgrade checks at the end:

https://github.com/openshift/origin/blob/c89105f8d6f67e087f4a06abba4122d9ea46ab3d/test/e2e/upgrade/upgrade.go#L178-L192

and 

https://github.com/openshift/origin/blob/c89105f8d6f67e087f4a06abba4122d9ea46ab3d/test/extended/util/disruption/disruption.go#L75-L90

(if we don't backport, the test will only run when the final upgrade version is >=4.15)

maybe @soltysh can confirm (or decline)